src/IntervalArithmetic.jl

```diff
@@ -68,7 +68,10 @@ export
     cancelminus, cancelplus, isbounded, isunbounded,
     .., @I_str, ±,
     pow, extended_div, nthroot,
-    setformat, @format
+    setformat, @format,
+    OverlapType, _both_empty, _first_empty, _second_empty, _before,
```

Comment:
if they are exported, maybe it would make more sense to call them `bothEmpty`, `firstEmpty` and so on?

Reply:
from today meeting: the issue was that `contains` is a julia function